packages/core/src/App/Containers/RealAccountSignup/choose-crypto-currency.jsx

```diff
@@ -0,0 +1,156 @@
+import { Field, Formik } from 'formik';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { FormSubmitButton, Text } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { reorderCurrencies } from '@deriv/shared';
+import { connect } from 'Stores/connect';
+import { CurrencyRadioButtonGroup, CurrencyRadioButton } from '@deriv/account';
+import './currency-selector.scss';
+
+const CRYPTO_CURRENCY_TYPE = 'crypto';
+
+const ChooseCryptoCurrency = ({
+    account_list,
+    available_crypto_currencies,
+    closeRealAccountSignup,
+    continueRouteAfterChooseCrypto,
+    currency_title,
+    legal_allowed_currencies,
+    openRealAccountSignup,
+    switchAccount,
+    should_show_all_available_currencies,
+    setShouldShowAllAvailableCurrencies,
+}) => {
+    React.useEffect(() => {
+        return () => setShouldShowAllAvailableCurrencies(false);
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
It's better to avoid disabling `react-hooks/exhaustive-deps`. I think adding `setShouldShowAllAvailableCurrencies` to the list of dependencies should be enough to avoid the warning.